
--- sorcer.ttl.old	2013-02-02 02:02:56.640992000 +0000
+++ sorcer.ttl	2013-02-02 02:04:39.537132413 +0000
@@ -6,12 +6,14 @@
 @prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix units: <http://lv2plug.in/ns/extensions/units#> .
+@prefix ui:    <http://lv2plug.in/ns/extensions/ui#> .
 <http://www.openavproductions.com/sorcer>
        a lv2:Plugin, lv2:InstrumentPlugin ;
        doap:name "Sorcer" ;
        lv2:binary <sorcer.so> ;
        lv2:optionalFeature epp:supportsStrictBounds ;
        lv2:optionalFeature lv2:hardRtCapable ;
+       ui:ui <http://www.openavproductions.com/sorcer/gui> ;
     lv2:port [
 	a lv2:InputPort ;
 	a lv2:ControlPort ;
