--- sorcer.ttl	2013-05-19 23:10:48.561731770 +0100
+++ sorcer.ttl.new	2013-05-19 23:10:04.753235000 +0100
@@ -6,14 +6,17 @@
 @prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix units: <http://lv2plug.in/ns/extensions/units#> .
+@prefix ui:    <http://lv2plug.in/ns/extensions/ui#> .
 <http://www.openavproductions.com/sorcer>
        a lv2:Plugin, lv2:InstrumentPlugin ;
        doap:name "Sorcer" ;
        lv2:binary <sorcer.so> ;
        lv2:optionalFeature epp:supportsStrictBounds ;
        lv2:optionalFeature lv2:hardRtCapable ;
+       lv2:optionalFeature ui:noUserResize ;
        doap:maintainer [ foaf:name "OpenAV Productions" ] ;
        doap:description "Wavetable Synth" ;
+       ui:ui <http://www.openavproductions.com/sorcer/gui> ;
     lv2:port [
 	a lv2:InputPort ;
 	a lv2:ControlPort ;
