--- sorcer.ttl.orig	2013-05-11 16:19:08.422244792 +0100
+++ sorcer.ttl	2013-05-11 16:23:30.442215720 +0100
@@ -6,6 +6,7 @@
 @prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs:  <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix units: <http://lv2plug.in/ns/extensions/units#> .
+@prefix ui:    <http://lv2plug.in/ns/extensions/ui#> .
 <http://www.openavproductions.com/sorcer>
        a lv2:Plugin, lv2:InstrumentPlugin ;
        doap:name "Sorcer" ;
@@ -14,6 +15,7 @@
        lv2:optionalFeature lv2:hardRtCapable ;
        doap:maintainer [ foaf:name "OpenAV Productions" ] ;
        doap:description "Wavetable Synth" ;
+       ui:ui <http://www.openavproductions.com/sorcer/gui> ;
     lv2:port [
 	a lv2:InputPort ;
 	a lv2:ControlPort ;
